zeq/event.cpp

```diff
@@ -13,49 +13,64 @@ Event::Event( const uint128_t& type )
     : _impl( new detail::Event( type ))
 {}
 
+#ifndef BOOST_NO_CXX11_RVALUE_REFERENCES
 Event::Event( Event&& rhs )
     : _impl( rhs._impl )
 {
     rhs._impl = 0;
 }
+#endif
 
 Event::~Event()
 {
     delete _impl;
 }
 
+void Event::copy( const Event& rhs )
+{
+    _impl = new detail::Event( rhs.getType() );
+    LBASSERT(_impl);
```

Comment:
It was a check that I thought was safer, but you made a point with the bad_alloc. I'll remove the asserts.